TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/controller/OnDirectionPressedListener.java

```diff
@@ -0,0 +1,28 @@
+package com.novoda.tpbot.controller;
+
+import com.novoda.notils.logger.simple.Log;
+
+interface OnDirectionPressedListener {
```

Comment:
I gave it some thought and figured that it was not really a Direction pad as it included a button too. It isn't a game pad as they usually have the multiple buttons to the right hand side of the controller. I ended up renaming the view to `Joystick`. It gives directions and has a fire button right in the middle of the directions. 

What do you think? ebe17a8